util/io/src/worker.rs

```diff
@@ -68,11 +66,8 @@ impl Worker {
 		};
 		worker.thread = Some(thread::Builder::new().name(format!("IO Worker #{}", index)).spawn(
 			move || {
-				panic_handler.catch_panic(move || {
-					Worker::work_loop(stealer, channel.clone(), wait, wait_mutex.clone(), deleting)
-				}).unwrap()
-			})
-			.expect("Error creating worker thread"));
+				Worker::work_loop(stealer, channel.clone(), wait, wait_mutex.clone(), deleting)
```

Comment:
`panic = "abort"` might not be a good idea because every panic will then lead to stuff like rocksdb handles not being closed properly.